media/javascript/readthedocs-analytics.js

```diff
@@ -2,33 +2,40 @@
 // https://docs.readthedocs.io/en/latest/advertising-details.html#analytics
 
 
-// RTD Analytics Code
+// Skip analytics for users with Do Not Track enabled
```

Comment:
you can use something like this [script](https://github.com/mozilla/kitsune/blob/990a8b4a7482ff78b692aa4a3c2fcc1b541dd04b/kitsune/sumo/static/sumo/js/dnt-helper.js) to check if `do not track` is enabled

Reply:
That entire script is basically to handle an IE10 bug. I'm not sure it's worth the effort.

Reply:
I think it should be in a script or a function that we can call from any scipt. Regarding IE,  we may still support IE10.

Reply:
IE10 is [not supported by Microsoft](https://www.microsoft.com/en-us/WindowsForBusiness/End-of-IE-support) with a couple exceptions and it is a tiny fraction of our users (sub-0.1%). I don't think it is unreasonable to not support a privacy feature for users who are using a browser unsupported by the vendor. In addition, the "support" that the linked script offers is mostly just to mark IE10 as "unspecified" for DNT which for our purpose would be off.

I lean toward simplicity here.

Reply:
Understand. then we can keep `window.doNotTrack === '1' || navigator.doNotTrack === '1')` in a function and call it from everywhere!

Reply:
Actually, after testing this I'm reconsidering. It looks like IE11 on Windows 7 and Windows 8 set the DNT default to on.

Reply:
Considering that to use this script would mean marking IE11 and IE10 as having DNT as "unspecified" I'm leaning toward maybe just checking `navigator.doNotTrack === '1'` and that's it. This would mean that no versions of IE can opt-out of tracking. It would be supported in MS Edge, however.

Reply:
> we can keep window.doNotTrack === '1' || navigator.doNotTrack === '1') in a function and call it from everywhere!

Not very easily actually. `readthedocs-analytics.js` is loaded on the docs pages and should not have any outside dependencies apart from `READTHEDOCS_DATA`.

Reply:
I updated the PR to just check `navigator.doNotTrack === '1'`.